---
title: Mapping structure and function in microbial communities
summary: What principles govern the mapping from the genomic structure to the metabolic activity of microbial communities? 
tags:
date: "2020-09-30T00:00:00Z"

image:
  caption: 
  focal_point: Smart


---

The structure, or "parts list," of microbial communities is composed of numerous diverse populations, each responding to different environmental cues, interacting with other populations, and shaping the chemical surroundings. Because of this complexity, it would appear that the relationship between community structure and collective metabolic function is hopelessly intractable. But what if only a small amount of the community "parts list" were relevant to collective metabolic activity? Discovering what principles govern the mapping from the genomic structure to the metabolic activity of microbial communities would give us great insight into how communities organize, operate, and evolve.

In this project, we demonstrated that the metabolite dynamics of a microbial community can be predicted from a small number of genes each member of the community possesses. To do this, we developed an experimental and statistical approach that leveraged natural variation over a collection of diverse bacterial strains to map genotypes to community-level metabolic phenotypes. This work illustrates that key metabolic genes can be used to predict community-level behavior, which opens the door to predicting metabolic fluxes in natural environments from metagenomic sequencing, designing denitrifying bacterial communities for use in wastewater treatment, and discovering how genome evolution impacts metabolic phenotypes.

Related manuscript:  
Genomic structure predicts metabolite dynamics in microbial communities  
Karna Gowda, Derek Ping, Madhav Mani, and Seppe Kuehn  
*Cell* **185**, (2022). doi: [10.1016/j.cell.2021.12.036](https://doi.org/10.1016/j.cell.2021.12.036)
